verl/trainer/main_ppo.py

```diff
@@ -146,10 +146,20 @@ def main_task(config):
         raise NotImplementedError
 
     compute_score = get_custom_reward_fn(config)
-    reward_fn = reward_manager_cls(tokenizer=tokenizer, num_examine=0, compute_score=compute_score)
+    reward_fn = reward_manager_cls(tokenizer=tokenizer,
+                                   num_examine=0,
+                                   compute_score=compute_score,
+                                   reward_fn_key=config.data.reward_fn_key,
+                                   max_resp_len=config.data.max_response_length,
+                                   overlong_buffer_cfg=config.custom_reward_function.overlong_buffer)
```

Comment:
I'm afraid that this makes the reward manager too specific to DAPO

Reply:
Reward Manager should be able to be pluggable and DAPO can implement its own reward in the recipe folder